3DCalendar/ViewController.swift

```diff
@@ -14,6 +14,8 @@ class ViewController: UIViewController, ARSCNViewDelegate {
 
     @IBOutlet var sceneView: ARSCNView!
     
+    let viewToAdd = CalendarView(frame: CGRect(x: 100, y: 100, width: 500, height: 550))
```

Comment:
Is there a way to have to just be the size of the node itself? Or does it need an explicit size?